packages/twenty-front/src/modules/command-menu/pages/page-layout/components/CommandMenuPageLayoutTabSettings.tsx

```diff
@@ -0,0 +1,126 @@
+import { CommandGroup } from '@/command-menu/components/CommandGroup';
+import { CommandMenuItem } from '@/command-menu/components/CommandMenuItem';
+import { CommandMenuList } from '@/command-menu/components/CommandMenuList';
+import { SidePanelHeader } from '@/command-menu/components/SidePanelHeader';
+import { useCommandMenu } from '@/command-menu/hooks/useCommandMenu';
+import { usePageLayoutIdFromContextStoreTargetedRecord } from '@/command-menu/pages/page-layout/hooks/usePageLayoutFromContextStoreTargetedRecord';
+import { useDeletePageLayoutTab } from '@/page-layout/hooks/useDeletePageLayoutTab';
+import { useMovePageLayoutTab } from '@/page-layout/hooks/useMovePageLayoutTab';
+import { useUpdatePageLayoutTab } from '@/page-layout/hooks/useUpdatePageLayoutTab';
+import { pageLayoutDraftComponentState } from '@/page-layout/states/pageLayoutDraftComponentState';
+import { pageLayoutTabSettingsOpenTabIdComponentState } from '@/page-layout/states/pageLayoutTabSettingsOpenTabIdComponentState';
+import { SelectableListItem } from '@/ui/layout/selectable-list/components/SelectableListItem';
+import { useRecoilComponentState } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentState';
+import { useRecoilComponentValue } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValue';
+import { useTheme } from '@emotion/react';
+import { t } from '@lingui/core/macro';
+import { isNonEmptyString } from '@sniptt/guards';
+import { isDefined } from 'twenty-shared/utils';
+import {
+  IconAppWindow,
+  IconChevronLeft,
+  IconChevronRight,
+  IconTrash,
+} from 'twenty-ui/display';
+
+export const CommandMenuPageLayoutTabSettings = () => {
+  const theme = useTheme();
+  const { closeCommandMenu } = useCommandMenu();
+  const { pageLayoutId } = usePageLayoutIdFromContextStoreTargetedRecord();
+
+  const draft = useRecoilComponentValue(
+    pageLayoutDraftComponentState,
+    pageLayoutId,
+  );
+  const [openTabId, setOpenTabId] = useRecoilComponentState(
+    pageLayoutTabSettingsOpenTabIdComponentState,
+    pageLayoutId,
+  );
+
+  const { moveLeft, moveRight } = useMovePageLayoutTab(pageLayoutId);
+  const { deleteTab } = useDeletePageLayoutTab(pageLayoutId);
+  const { updatePageLayoutTab } = useUpdatePageLayoutTab(pageLayoutId);
+
+  if (!isDefined(openTabId)) {
+    return null;
+  }
+
+  const tabsSorted = [...draft.tabs].sort((a, b) => a.position - b.position);
```

Comment:
Nitpick: You can use the new [`Array.toSorted`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted) function to sort an array without updating the original reference.